libs/design/tabs/src/tabs/tabs.component.scss

```diff
@@ -1,6 +1,10 @@
 :host(.daff-tabs) {
 	$root: '.daff-tabs';
 	max-width: 100%;
+
+	a {
+		text-decoration: none;
+	}
```

Comment:
@xelaint I can't get this to remove the underline from the tabs, can you fix?

Reply:
fixed